modules/osd-create-cluster-gcp-account.adoc

```diff
@@ -0,0 +1,105 @@
+.. Select the *On-Demand* subscription  type.
+.. From the dropdown menu, select *Google Cloud Marketplace*.
+.. Select the *Customer Cloud Subscription* infrastructure type.
+.. Click *Next*.
+. On the *Cloud provider* page, read the provided prerequisites and the Google terms and conditions. Add your service account key.
+.. Click the *Review Google Terms and Agreements* link.
+.. If you agree to the Google terms and agreements and want to proceed, continue completing the steps in the {product-title} cluster create wizard.
+.. Add your service account key.
++
+[NOTE]
+====
+For more information about service account keys, click the information icon located next to *Service account key*.
+====
+.. Click *Next*.
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity for the cluster. For more information, see the _Storage_ section in the {product-title} service definition.
+.. Specify the number of *Load balancers* that you require for your cluster. For more information, see the _Load balancers_ section in the {product-title} service definition.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. This option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Click *Next*.
+
+. On the *Machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes, but you cannot change the compute node instance type in a machine pool. For clusters that use the CCS model, you can add machine pools after installation that use a different instance type. The number and types of nodes available to you depend on your {product-title} subscription.
```

Comment:
For this sentence:

1.  `you cannot change the compute node instance type in a created machine pool` could be more strict. 
2. No need `For clusters that use the CCS model`, either ccs or non-ccs, the customer can add the machine pool.
3. `use a customized instance type` could be better.